S21/cihanozluk/ToLower/ToLower/UpperToLower.cs

```diff
@@ -0,0 +1,76 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+namespace ToLower
+{
+    public class UpperToLower
+    {
+
+        public UpperToLower(string word, string language)
+        {
+            if (string.IsNullOrEmpty(language))
+            {
+                throw new ArgumentException($"'{nameof(language)}' cannot be null or empty.", nameof(language));
+            }
+            Word = word ?? throw new ArgumentNullException(nameof(word));
+            Language = language;
+
+        }
+
+
+        public string Word { get; }
+
+        public string Language { get; }
+
+
+        private string GetToLower() => Word.ToLower(new System.Globalization.CultureInfo(Language));
+
+        public string ToGetLower()
+        {
+            List<Alphabet> alphabets = new List<Alphabet>();
+            alphabets.Add(new Alphabet("tr", "ABCÇDEFHĞHIİJKLMNOÖPQRSŞTUÜVWXYZ", "abcçdefgğhıijklmnoöpqrsştuüvwxyz"));
+            alphabets.Add(new Alphabet("el", "ΑΒΓΔΕΖΗΘΙΚΛΜΝΞΟΠΡΣΤΥΦΧΨΩ", "αβγδϵζηθικλμνξοπρστυϕχψω"));
+            alphabets.Add(new Alphabet("ga", "ABCDEFGHILMNOPRSTUÁÉÍÓÚ", "abcdefghilmnoprstuáéíóú"));
+            alphabets.Add(new Alphabet("ga-IE", "ABCDEFGHILMNOPRSTUÁÉÍÓÚ", "abcdefghilmnoprstuáéíóú"));
+
+            if (Language != "tr" && Language != "el" && Language != "ga" && Language != "ga-IE")
+                return GetToLower();
+            else
+            {
+                var lowerWord = new StringBuilder();
+                var str = alphabets.Find(r => r.Language == Language);
```

Comment:
This variable could be named something meaningful

Reply:
you mean str rhigt